### ISSUE TITLE
The future of this project

### ISSUE THREAD

**Issue by nicklaw5** [nicklaw5/stryve#60]
This issue serves as a chalkboard to throw around some ideas on what this project should or could become if development continued.

### Current State
In it's current state there are many issues. To mention just a few:
- the project is split into four different repositories including this one, which make it difficult to maintain:
  - [stryve](https://github.com/nicklaw5/stryve) - the web and desktop GUI client
  - [stryve-api-php](https://github.com/nicklaw5/stryve-api-php) - the API server
  - [stryve-api-client](https://github.com/nicklaw5/stryve-api-client) - the API client for interacting with the API server
  - [stryve-socket-server](https://github.com/nicklaw5/stryve-socket-server) - the socket server that handles at the real-time communication between clients
- the web app is very slow to load due to the large [webpack](http://webpack.github.io/) build file
- only Chrome browser was supported
- the UI is borrowed from [Discord](https://discordapp.com)
- no tests were written to support the code itself
- no linting was used to maintain code structure
- no mobile support was initially intended
- client authentication is thin and insecure
- user-to-user chat is not coded very well and connecting with others is obscure in it current state

### Future Proposal
*Ideas are still formulating...*

**Comment by Puyodead1** [comment]
Please keep this going. This is the best!

**Comment by mastercho** [comment]
Is project abandoned?

**Comment by nicklaw5** [comment]
@mastercho yes this project has been abandoned. May I suggest you look at [Rocket Chat](https://rocket.chat/) as an alternative.

**Comment by mastercho** [comment]
@nicklaw5 tried found couple security breaches + not well developed so its bad

**Comment by kylesmith63** [comment]
I kept hoping this project would pick up development again
